### PR TITLE
[ Refactor ] Format and Lint / Fix Typings

### PR DESCRIPTION
**Pull request by posaune0423**
- [x] setup formatter and linter
- [x] Add workflow for frontend
- [x]  fix typing

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -194,131 +184,142 @@ function App() {
...
+                    <Route path='/governance' element={<Governance />} />
+                    <Route path='/new-proposal' element={<NewProposal />} />
+                    <Route path='/proposal/:id' element={<ProposalDetails />} />
                 </Routes>
             </div>
             <ToastContainer
-                position="top-right"
-                className={'w-full max-w-[420px] min-w-[200px] z-999 top-12'}
+                position='top-right'
+                className={'top-12 z-50 w-full min-w-[200px] max-w-[420px]'}
```
Comment (0xshora) [inline src/App.tsx]: z-50 : toast is behind of Create A New Proposal Popup.
z-999 might be too big, but 50 is not enough.

**Comment by 0xshora** [comment]
This work is so impressive!!
When I try this version, the speed of loading/refreshing is so much faster!

but, activating Reset To White is not working...

File: src/App.tsx (hunk 1 above)
Comment (posaune0423) [reply src/App.tsx]: Okey, I added custom zIndex `z-999` so that toast properly shown!

**Comment by 0xshora** [comment]
Thanks, it's working.
The only issue we have now is this:
image

**Pull request merged**